steps/steps.py

```diff
@@ -0,0 +1,96 @@
+from behave import *
+from selenium import webdriver
+
+
+import environment
+import time
+
+
+class Calculator(object):
+    @step('that the Calculator app is loaded')
+    def app_is_loaded(context):
+        context.browser.get(
+            'http://seleniumsimplified.com/testpages/calculate.php'
+        )
+
+    @step('there is no current value in the calculator')
+    def check_current_value(context):
+        first_number_input = context.browser.find_element_by_xpath(
+            '//input[@id="number1"]'
+        )
+        second_number_input = context.browser.find_element_by_xpath(
+            '//input[@id="number2"]'
+        )
+
+        if first_number_input.get_attribute("value") != '':
+            first_number_input.clear()
+            if second_number_input.get_attribute("value") != "":
+                second_number_input.clear()
+        assert (
+            first_number_input.get_attribute("value") and
+            second_number_input.get_attribute("value")
+            ) == ''
```

Comment:
Obawiam się, że ta asercja nie działa tak jak się spodziewasz. Musiałaby wyglądać tak:

    assert (
        first_number_input.get_attribute('value') == '' and
        second_number_input.get_attribute('value' == '')
        )

a bardziej pythonowo:

    assert (
        not first_number_input.get_attribute('value') and 
        not second_number_input.get_attribute('value')
    )

To co napisałaś działa tak: najpierw robi operację logiczną w nawiasie `(jakis_string_1 and jakis_string_2)`, a to w pythonie działa tak, że jeśli pierwszy string będzie miał jakąkolwiek wartość, to zwrócony zostanie drugi string. Jeśli pierwszy string będzie pusty, to drugi nawet nie jest sprawdzany - zostanie zwrócony ten pierwszy pusty string. Sama spróbuj w interpreterze pythona:

    >>> 'costam1' and ''
    ''
    >>> 'costam1' and 'costam2'
    'costam2'
    >>> '' and 'costam2'
    ''
    >>> '' and ''
    ''

... no i dopiero po tej operacji logicznej z nawiasu, wynik jest porównywany do pustego stringa, a to oznacza, że asercja będzie spełniona nawet jeśli tylko jedno z pól było puste, a chyba nie o to Ci chodziło? :)